doc/DeveloperGuide.md

```diff
@@ -58,9 +58,12 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
+`* * *` | user | exit the address book | use other applications in my device
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | edit a person's contact detail | update changes easily
```

Comment:
May be `update person's contact detail without executing àdd and delete commands.` is clearer.